### DISCUSSION TITLE
Content about Perl 1

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/Perl-1.0#43] [Q&A] [answered]
Is there any content created about Perl 1?

**Comment by Stagyrite** [comment, marked as answer]
Yes, there is. First, I'd like to mention [the original README file](https://raw.githubusercontent.com/Stagyrite/Perl-1.0/main/README). It's from the year 1987 when the Perl 1 was created. Second, you might want to see [a README.chastai file](https://github.com/Stagyrite/Perl-1.0/blob/main/README.chastai). This is from the year 2014. In the year 2024, [a new site](https://stagyrite.github.io/Perl-1.0/ "Stagyrite | Perl 1") has been created. It provides my Perl Geek Code, as well as the discussions. Nevertheless, you should take a look at [the code](https://github.com/Stagyrite/Perl-1.0 "Stagyrite/Perl-1.0: 🐪 Perl Kit, Version 1.0"), if interested. Knowledge of Perl programming and compilers would be helpful in this particular task.
